tests/src/test/java/alluxio/client/rest/S3ClientRestApiTest.java

```diff
@@ -56,20 +58,24 @@
 import com.google.common.io.BaseEncoding;
 import org.apache.commons.codec.binary.Hex;
 import org.apache.commons.io.IOUtils;
+import org.apache.hadoop.hbase.util.Strings;
```

Comment:
Do you have to involve hbase which is a new dependency for alluxio

Reply:
Oh thanks I didn't catch this. My guess is no, so we can change that.

Reply:
PR link: https://github.com/Alluxio/alluxio/pull/16039